assets/src/styles/blocks/core-overrides/_columns.scss

```diff
@@ -1,5 +1,5 @@
-.is-style-mobile-carousel {
-  @include mobile-only {
+.wp-block-columns.is-not-stacked-on-mobile.is-style-mobile-carousel {
+  @media (max-width: 767px) {
```

Comment:
I think we have a mixin `@mixin small-and-less` that fits this ? not sure if we should use it/multiply mixins but it's there

Reply:
Good point, I didn't know it existed! Thanks, updated 👍